## Programming Assignment - Part 6

### 6 Statistic Tests 

In [1]:
import part6_simstudy

#### 6.2 Verification

In [2]:
part6_simstudy.task_6_2_1()

Alpha; 0.1 Bins; 1 Chi2;3.286766830846552e-11 Chi2 Table;nan Null hypothesis is not rejected: the samples follow the standard normal distribution
Alpha; 0.1 Bins; 2 Chi2;1.0000005733363402 Chi2 Table;nan Null hypothesis is not rejected: the samples follow the standard normal distribution
Alpha; 0.1 Bins; 5 Chi2;0.533180950263032 Chi2 Table;2.705543454095417 Null hypothesis is not rejected: the samples follow the standard normal distribution
Alpha; 0.1 Bins; 10 Chi2;1.2489716570240013 Chi2 Table;4.605170185988092 Null hypothesis is not rejected: the samples follow the standard normal distribution
Alpha; 0.1 Bins; 20 Chi2;6.946523219153302 Chi2 Table;10.64464067566842 Null hypothesis is not rejected: the samples follow the standard normal distribution
Alpha; 0.1 Bins; 30 Chi2;3.444691066295358 Chi2 Table;13.36156613651173 Null hypothesis is not rejected: the samples follow the standard normal distribution
Alpha; 0.05 Bins; 1 Chi2;3.286766830846552e-11 Chi2 Table;nan Null hypothesis is no

#### 6.3 Analysis and General Questions

##### Task 6.3.1: Chi Square Test

* If we compare different alpha values e.g. Alpha; 0.1 Bins; 20, Chi2 Table;10.64464067566842 and Alpha; 0.01 Bins; 20 Chi2 Table;16.81189382977092, the rejection probability increases with higher significance alpha values.
* Lower bin values dont provide Chi2 Table result which increase rejection probability and also, with higher bin numbers decrease rejection probability.
* Comparing distributions with different mean and variance increases rejection probability.